src/list/components/item-list.css

```diff
@@ -5,13 +5,22 @@
 .empty {
   text-align: center;
   white-space: pre-wrap;
-  padding: 40px;
+  padding: 40px 20px;
```

Comment:
technically this deviates from Zeplin, but actual rendering (on my machine 🙃 ) I think better matches the intent.